config/routes.rb

```diff
@@ -9,6 +9,8 @@
   root 'welcome#index'
   get 'game_day', to: 'game_day#index', as: 'game_day'
   get 'drafts/:draft_id/picks', to: 'drafts#picks', as: 'draft_picks'
+  get 'rules', to: 'rules#index', as: 'rules'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.